.github/workflows/cd.yml

```diff
@@ -64,17 +64,21 @@ jobs:
 
   deploy-k8s-service:
     name: Deploy to Kubernetes
+    runs-on: ubuntu-22.04
     needs:
       - tag
       - build-push-docker
-    uses: bitwarden/devops/.github/workflows/_deploy_k8s_service.yml@b9c4c99a4cb0330465af8b30c9a7a4e8202dec25
-    with:
-      image-names: billing-relay
-      environment: Mothership
-      overlay: mship-cloud-netnew
-      repository: billing-relay
-      tag: ${{ needs.tag.outputs.git-tag }}
-    secrets: inherit
+
+    steps:
+      - name: Dispatch deployment
+        run: >
+          gh workflow run _deploy_k8s_service
```

Comment:
ℹ️ There's one of these without the underscore used in other places, but has no mothership support. Should I just stop using this?

Reply:
I would continue to use `_deploy_k8s_service` since that's how it was working before, however, I'm not sure if `gh workflow run` works if there isn't a `workflow_dispatch` trigger in the workflow file.